[2023-02-26T13:39:18.553+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: my_dag.inaccurate manual__2023-02-26T13:39:14.885718+00:00 [queued]>
[2023-02-26T13:39:18.560+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: my_dag.inaccurate manual__2023-02-26T13:39:14.885718+00:00 [queued]>
[2023-02-26T13:39:18.560+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-02-26T13:39:18.561+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-02-26T13:39:18.562+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-02-26T13:39:18.570+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): inaccurate> on 2023-02-26 13:39:14.885718+00:00
[2023-02-26T13:39:18.574+0000] {standard_task_runner.py:55} INFO - Started process 8830 to run task
[2023-02-26T13:39:18.577+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'inaccurate', 'manual__2023-02-26T13:39:14.885718+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpe88crn4x']
[2023-02-26T13:39:18.578+0000] {standard_task_runner.py:83} INFO - Job 23: Subtask inaccurate
[2023-02-26T13:39:18.630+0000] {task_command.py:388} INFO - Running <TaskInstance: my_dag.inaccurate manual__2023-02-26T13:39:14.885718+00:00 [running]> on host 8aab065f5efa
[2023-02-26T13:39:18.674+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=inaccurate
AIRFLOW_CTX_EXECUTION_DATE=2023-02-26T13:39:14.885718+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-26T13:39:14.885718+00:00
[2023-02-26T13:39:18.675+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-02-26T13:39:18.676+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "echo 'inaccurate'"]
[2023-02-26T13:39:18.687+0000] {subprocess.py:86} INFO - Output:
[2023-02-26T13:39:18.689+0000] {subprocess.py:93} INFO - inaccurate
[2023-02-26T13:39:18.689+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-02-26T13:39:18.710+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=inaccurate, execution_date=20230226T133914, start_date=20230226T133918, end_date=20230226T133918
[2023-02-26T13:39:18.750+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-02-26T13:39:18.764+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
